### PR TITLE
Added RavenJS to track production error

### PR DESCRIPTION
**Pull request by fish-sushi**
@kingsi-goderia 
Please add Solidity side error tracker by using Sentry.

just ask me about our Sentry account.

### CODE REVIEW

**Review by parkkoon — APPROVED** [review]
👍

File: app/javascripts/app.jsx [hunk 1]
```diff
@@ -31,6 +32,7 @@ if (!EnvChecker.isDev()) {
   });
   ReactGA.set({ page: window.location.pathname + window.location.search });
   ReactGA.pageview(window.location.pathname + window.location.search);
+  Raven.config('https://1c0693f3ef3e4aa5bcc6895c3848b2e4@sentry.io/190512').install();
```
Comment (am2rican5) [inline app/javascripts/app.jsx]: Is it ok to use this url in raw string instead of environment variable?

Comment (kingsi-goderia) [inline app/javascripts/app.jsx]: Is it ok to use this url in raw string instead of environment variable?

Comment (fish-sushi) [reply app/javascripts/app.jsx]: That will be better.
I'll fix it

**Review by c0smic0wl — APPROVED** [review]
🦉

**Review by kingsi-goderia — APPROVED** [review]
It looks good to me

**Pull request merged**